.github/workflows/fmu-tools.yml

```diff
@@ -29,28 +29,19 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
-    - name: Install numpy for Python 2
-      if: matrix.python-version == '2.7'
+    - name: Install requirements
       run: |
-        pip install numpy==1.16.6
+        pip install pip -U
+        pip install git+https://github.com/equinor/libecl
+        pip install git+https://github.com/equinor/fmu-ensemble
+        pip install .[tests]
```

Comment:
What do `pip install .[tests]` do?

Reply:
it is the same as `pip install .` but it also picks up "extras" for `tests`as declared in `setup.py` and installs those dependencies also.

Reply:
Nice: new to me